examples/reshare/src/dkg/actor.rs

```diff
@@ -170,71 +176,27 @@ where
         //
         // For the sake of the example, we assume a fixed set of contributors that can be selected
         // from.
-        let all_participants = active_participants
```

Comment:
I decided not to tackle this in this PR but it would be nice to simplify this logic.